connectors/connectors/day-trade/day-trade-get-connector/src/main/java/io/syndesis/connector/daytrade/DayTradeGetComponent.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2017 Red Hat, Inc.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
these `<p>`'s look wrong

Reply:
Sorry, not sure where this comes from. I haven't tuned the license manually (as I really never care about license formatting at first priority and tbh, I would be really ok to switch off the license check and do an automatic validation / adaption / verification once in a while ... I already spent to much time on making this plugin happy (instead of vice-versa as it should be).

For real code checks I have a different opinion ....

Nevertheless, I will investigate.